test/__fixtures__/page.tsx

```diff
@@ -174,3 +175,19 @@ export const createBlogPageData = (): BlogPageData => ({
     ],
   },
 });
+
+export const createGovernanceData = (): Page => ({
```

Comment:
I've noticed that apart from this, #1336, #1298, #1297 and #1296 use the exact same data for testing but use different function name. And I think it'll be created again for any new static page that we're going to add in issues like #1341.

I think we can keep a generic function for this type of PRs. Maybe we can change the function name to a generic one here, and then use it as an example for all other old / upcoming PRs.